### ISSUE TITLE
Add alt tag to inline images used on home, shot and myshots

### ISSUE THREAD

**Issue by punamdahiya** [mozilla-services/screenshots#4976]
See https://github.com/mozilla-services/screenshots/pull/4940#discussion_r221362791

**Comment by chenba** [comment]
All non-decorative images should have alt text.

**Comment by punamdahiya** [comment]
Tested with Voice Over Screen Reader on Safari, title attribute of button/link on my shots page - signin, share, download  and delete button get read after identifying tag as button or link. 

Only exception is favorite button (my shots) which is read as group  and should be updated to button instead of div

**Comment by punamdahiya** [comment]
Closing this as duplicate of #4667
